.nvmrc

```diff
@@ -0,0 +1 @@
+lts/hydrogen
```

Comment:
nit: this is a mutable target, similar to using `"react": "^18.9.0",` in a `package.json` but WITHOUT the benefit of a lockfile. On my teams we pin this to an explicit version.

Tools like nvm that read this and automatically switch versions will re-download Node.js for users upon traversing to this directory. A good thing, perhaps, but also noisy.

If this feels like too much of a burden to keep updating, disregard.

Reply:
TL;DR, we want to keep in sync that the prod version of Node is the same as the local one. IMHO, it makes error reporting and debugging somewhat easier by discarding Node.js versions of the equation.